enterprise_extensions/model_utils.py

```diff
@@ -1,28 +1,26 @@
 # -*- coding: utf-8 -*-
-from __future__ import (absolute_import, division,
-                        print_function)
+
 import time
-import numpy as np
-import scipy.stats as scistats
 
 import matplotlib.pyplot as plt
+import numpy as np
+import scipy.stats as scistats
 
 try:
```

Comment:
same extraneous `try`/`except` here.

Reply:
for some reason this is highlighting the wrong lines

Reply:
Got it!